ts/features/itwallet/common/store/reducers/preferences.ts

```diff
@@ -120,7 +120,15 @@ const reducer = (
       };
     }
     case getType(itwLifecycleStoresReset):
-      return { ...itwPreferencesInitialState };
+      // When the wallet is being reset, we need to persist only the preferences:
+      // - claimValuesHidden
+      // - isL3Enabled
+      const { claimValuesHidden, isL3Enabled } = state;
+      return {
+        ...itwPreferencesInitialState,
+        claimValuesHidden,
+        isL3Enabled
```

Comment:
Is it correct to persist the L3-enabled status after a reset, when the eID/PID is removed?

Reply:
Yes, it is correct. It is not linked with the presence of the eID/PID. It is just a flag that enables the "L3 Mode" whenever the users choose the CIE+PIN auth method.
Also consider that this will be replaced with the remote FF, so it is a temporary solution and the logic may change.

Reply:
Btw, you just made me not that the flag should be applied only when we chose CIE+PIN. I've updated the logic here: https://github.com/pagopa/io-app/pull/6875/commits/dfa3282092556918c505354b9987257c5c21ed9e